### DISCUSSION TITLE
Welcome to fasthep-flow Discussions!

### DISCUSSION THREAD

**Discussion by kreczko** [FAST-HEP/fasthep-flow#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by YeungOnion** [comment]
@kreczko hey, wondering if there's anything I can help out with for the migration to prefect, package restructure, or elsewhere in the project. Alternatively, let me know if too early in the design/decision space for open contributions to be best.

atm I'm
- keeping an eye out on fasthep-carpenter's [issues](https://github.com/FAST-HEP/fasthep-carpenter/issues)
- willing to test and write tests for CLI
- working on docs

**Reply by kreczko** [reply]
I've now added some issues and I will be adding more over the next few weeks - if there is something you are interested, feel free to comment in the issue and start on it.
We have now a first version in the new system, I am going to bring more in bit-by-bit from the experimental branch from https://github.com/FAST-HEP/fasthep-carpenter/tree/kreczko-1.0.0a1/src/fasthep_carpenter
